Dockerfile

```diff
@@ -10,13 +10,53 @@ COPY vendor/php-parser/composer.lock /usr/src/app/vendor/php-parser/
 
 COPY package.json /usr/src/app/
```

Comment:
Would it be possible to move copy all of the lock files and package files below the update and install steps? Otherwise, bumping gems will invalidate all of these layers